docs/index.md

```diff
@@ -15,45 +15,73 @@ Also, we are currently looking for a job. If your team needs software engineers,
 
-Run `ADMIN=YOUR_ADMIN_PASSWORD npm run server`. You will have to run this as `root` or use `sudo`. This will install required dependencies and start the server.
-* Installs `git` if system does not have it.
-* Installs `nodeJS` and `npm` if system does not have it.
-* Enable firewall port `3000` (for the admin page UI), `80` and `443`.
-* Installs application dependencies
-* For a list of things the script runs, [look here](https://github.com/garageScript/myProxy/blob/master/scripts/setup.sh)
+2. Clone the app: `git clone https://github.com/garageScript/myProxy.git`
 
-Exit from server `exit`
+3. Goto myProxy folder: `cd myProxy`
 
-Go to your server url: `http://your-server-ip-address:3000`. You will be prompted to put in your domain provider's API Key and Secret, [find out how here](https://github.com/Neilpang/acme.sh/wiki/dnsapi)
+4. Run Setup Script `./scripts/setup.sh`
+    This will installed require dependencies.
+    * Installs `nodeJS` and `npm` if system does not have it.
+    * Enable firewall port `3000` (for the admin page UI), `80` and `443`.
+    * Installs application dependencies
+    * For a list of things the script runs, [look here](https://github.com/garageScript/myProxy/blob/master/scripts/setup.sh)
+
+5. Run the App: `ADMIN=YOUR_ADMIN_PASSWORD npm run server` 
+> You can also run the app under your own defined port by setting a `PORT` environment variable
+
+6. Exit from server `exit`
+
+7. Go to your server url: `http://your-server-ip-address:3000`. You will be prompted to enter your admin password and your domain provider's API Key and Secret, [find out how here](https://github.com/Neilpang/acme.sh/wiki/dnsapi)
+
+8. All your domain names in that provider will show up. Click the `setup` button next to the domain you wish to setup (could take up to 4 minutes)
+
+9. After your domain is setup, you will be able to generate as many subdomain repository as you want! To do that:
+    1. Go to your server url:  `http://your-server-ip-address:3000`
+    2. Create a subdomain. IP and port are optional. You should see a git link that was created for you.
+    3. `git clone` the app, then build the app locally. [find out how here](##Build-Your-Local-App)  
+    4. When you are done, `git push origin master` and watch your app run in production!
 
-All your domain names in that provider will show up. Click the `setup` button next to the domain you wish to setup (could take up to 3 minutes)
 
-After your domain is setup, you will be able to generate as many subdomain repository as you want! To do that:
-1. Go to your server url:  `http://your-server-ip-address:3000`
-2. Create a subdomain. Ip and port are optional. You should see a git link that was created for you.
-3. `git clone` the app, then build the app locally.
-4. When you are done, `git push origin master` and watch your app run in production!
+## Building-Your-Local-App 
+1. In the terminal, go to your App folder.
```

Comment:
1. In the terminal, clone and enter your app folder: `git clone <your fullDomain repo>`
2. enter your repo: `cd <your fullDomain folder>`
3. run `npm init -y`
...